enterprise/backend/test/metabase_enterprise/serialization/v2/extract_test.clj

```diff
@@ -897,7 +897,15 @@
             (is (= #{[{:model "Database"   :id "My Database"}
                       {:model "Table"      :id "Schemaless Table"}
                       {:model "Field"      :id "Some Field"}]}
-                   (set (serdes.base/serdes-dependencies ser))))))))))
+                   (set (serdes.base/serdes-dependencies ser)))))))
+      (testing "extract-metabase behavior"
+        (testing "without :include-field-values"
+          (is (= #{}
+                 (by-model "FieldValues" (extract/extract-metabase {})))))
+        (testing "with :include-field-values true"
+          (let [models (->> {:include-field-values true} extract/extract-metabase (map (comp :model last :serdes/meta)))]
```

Comment:
Can you use `by-model` here too?

Reply:
No.

Reply:
Not to be short 😆 but `by-model` returns a set of IDs. It just happens to work above.